components/BottomNavigation/tests/unit/Theming/MDCBottomNavigationThemingTests.m

```diff
@@ -0,0 +1,180 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialBottomNavigation+Theming.h"
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialColorScheme.h"
+#import "MaterialContainerScheme.h"
+#import "MaterialShadowElevations.h"
+#import "MaterialTypography.h"
+#import "MaterialTypographyScheme.h"
+
+static const CGFloat kUnselectedPrimaryAlpha = 0.74f;
+static const CGFloat kUnselectedSurfaceAlpha = 0.6f;
+
+@interface MDCBottomNavigationThemingTests : XCTestCase
+@property(nonatomic, strong, nullable) MDCBottomNavigationBar *bottomNavigationBar;
+@end
+
+@implementation MDCBottomNavigationThemingTests
+
+- (void)setUp {
+  [super setUp];
+
+  self.bottomNavigationBar = [[MDCBottomNavigationBar alloc] init];
+  UITabBarItem *tabBarItem1 = [[UITabBarItem alloc] initWithTitle:@"Home"
+                                                            image:[UIImage imageNamed:@"Home"]
```

Comment:
I don't see these named images ("Home", "Email", "Favorite") in the Bottom Navigation Unit Test resources. Were these added by accident from an example copy-paste?

Please either add these resources to the target or use a fake image similar to how `BottomNavigationItemViewTests` does it.

Reply:
Done